### DISCUSSION TITLE
How to verify that a leaf of type "empty" is in the Input of an ActionRequest

### DISCUSSION THREAD

**Discussion by sebastien-guay** [freeconf/restconf#20] [Q&A] [answered]
Hi, I'm Sebastien and I work with Patrick Dumais. Our yang model has a leaf of type `empty`. I would like to know how I can verify that this leaf data is in the `ActionRequest.Input`?

Calling req.Input.GetValue() will always return `nil` whatever the leaf is in the data or not.

Regards,

Sebastien

**Comment by dhubler** [comment, marked as answer]
Hmm, I looked and don't see a way. Can you open a bug?

**Reply by sebastien-guay** [reply]
Here is the issue: https://github.com/freeconf/restconf/issues/21
